staging/src/k8s.io/component-base/version/verflag/verflag.go

```diff
@@ -28,26 +28,24 @@ import (
 	"k8s.io/component-base/version"
 )
 
-type versionValue int
+type VersionValue string
 
 const (
-	VersionFalse versionValue = 0
-	VersionTrue  versionValue = 1
-	VersionRaw   versionValue = 2
+	VersionFalse VersionValue = "false"
+	VersionTrue  VersionValue = "true"
+	VersionRaw   VersionValue = "raw"
 )
 
-const strRawVersion string = "raw"
-
-func (v *versionValue) IsBoolFlag() bool {
+func (v *VersionValue) IsBoolFlag() bool {
 	return true
 }
 
-func (v *versionValue) Get() interface{} {
-	return versionValue(*v)
+func (v *VersionValue) Get() interface{} {
+	return VersionValue(*v)
 }
 
-func (v *versionValue) Set(s string) error {
-	if s == strRawVersion {
+func (v *VersionValue) Set(s string) error {
+	if s == string(VersionRaw) {
```

Comment:
The following might be simpler:
```go
switch s {
  case VersionFalse, VersionTrue, VersionRaw:
    *v = s
  default:
    return fmt.Errorf("unrecognized version option: %s", s)    
}
return nil
```

Reply:
That is true, but I didn't want to change the existing functionality which utilizes `ParseBool()`. I don't know if anybody's using it, but, as a consequence the `--version` flag also accepts values like `True`, `TRUE`, `1`, `t` etc.

I can change that if you think it's desirable in the scope of this PR. WDYT?

Reply:
I'd rather keep it as is. That might break some users if we change it.